server/src/main/java/org/apache/iotdb/db/qp/logical/crud/InOperator.java

```diff
@@ -55,7 +56,7 @@
    * @param path path
    * @param values values
    */
-  public InOperator(int tokenIntType, Path path, boolean not, Set<String> values) {
+  public InOperator(int tokenIntType, PartialPath path, boolean not, Set<String> values) {
```

Comment:
If the PartialPath extends the Path, this is no need to change.

Reply:
fixed